File: tests/fixtures/sample.rb
class Calculator
  def add(a, b)
    a + b
  end

  def multiply(a, b)
    a * b
  end
end

module MathHelpers
  def self.factorial(n)
    return 1 if n <= 1
    n * factorial(n - 1)
  end
end

File: tests/fixtures/sample.scala
package sample

class Calculator {
  def add(a: Int, b: Int): Int = a + b
  def multiply(a: Int, b: Int): Int = a * b
}

trait Printable {
  def prettyPrint(): String
}

object MathUtils {
  def factorial(n: Int): Int =
    if (n <= 1) 1 else n * factorial(n - 1)
}
